pkg/operator/controllers/genevalogging/genevalogging.go

```diff
@@ -23,8 +23,9 @@ import (
 )
 
 const (
-	GenevaCertName = "gcscert.pem"
-	GenevaKeyName  = "gcskey.pem"
+	GenevaCertName  = "gcscert.pem"
+	GenevaKeyName   = "gcskey.pem"
+	acrDomainSuffix = ".azurecr.io" // TODO: retrieve dynamically
```

Comment:
sooner or later we'll need to pass AZURE_ENVIRONMENT as an environment variable to the operator and pick it up.  Maybe by s/ACRName/ACRDomain/ we can put that moment off.

Reply:
AZURE_ENVIRONMENT can be determined by reading cloud provider configuration. Do we want to still pass it?